street_lights/index.html

```diff
@@ -1,12 +1,62 @@
-<!DOCTYPE html>
 <html>
   <head>
     <meta charset="utf-8">
     <title>Światła drogowe</title>
   </head>
-  <body>
+    <body>
+  
     <div id="streetLights">
```

Comment:
No dobra niech i tak bedzie, teraz dziala ale to jest blad semantyczny
nie mozesz uzywac id do wielu elementów. Zmień to na coś innego co bedzie mozna uzywac w tej sytuacji. Po co Ci to id ?